ui/raidboss/data/05-shb/raid/e8n.txt

```diff
@@ -4,4 +4,105 @@
 hideall "--Reset--"
 hideall "--sync--"
 
+# -ii 4D58 4DD3 4DDF 4DE0 4DF3 4DF5 4E0B 4E0D 4E10 4E11 4E18 4FCA
+
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# Opening block
+0.0 "--sync--" sync /:Engage!/ window 0,1
+1.4 "--sync--" sync /:Shiva:4DD3:/ window 1.4,2
+15.1 "Absolute Zero" sync /:Shiva:4DD7:/ window 15,5
+27.2 "Redress" sync /:Shiva:4F4F:/
+32.4 "Shining Armor" sync /:Shiva:4DF1:/
+40.4 "Axe Kick/Scythe Kick" sync /:Shiva:4DE[23]:/
+52.5 "Redress" sync /:Shiva:4F4E:/
+57.6 "Frost Armor" sync /:Shiva:4DF0:/
+65.6 "Biting Frost/Driving Frost" sync /:Shiva:4DD[BC]:/
+78.8 "Double Slap" sync /:Shiva:4DDA:/
+92.0 "Diamond Frost" sync /:Shiva:4DE1:/
+100.2 "Frigid Water/Frigid Stone" # sync /:Shiva:4E(?:08|66):/
+103.1 "Icicle Impact" sync /:Shiva:4E0A:/
+110.2 "Frigid Eruption" # sync /:Shiva:4E09:/
+111.2 "Icicle Impact" sync /:Shiva:4E0A:/
+112.3 "Frigid Eruption" # sync /:Shiva:4E09:/
+114.3 "Frigid Eruption" # sync /:Shiva:4E09:/
+120.1 "Heavenly Strike" sync /:Shiva:4DD8:/
+131.2 "Mirror, Mirror" sync /:Shiva:4DD4:/
+143.4 "Driving Frost/Biting Frost" sync /:Shiva:4DD[BC]:/
+148.5 "Reflected Frost" sync /:Frozen Mirror:4DF[EF]:/
+160.6 "Redress" sync /:Shiva:4F4F:/
+165.7 "Shining Armor" sync /:Shiva:4DF1:/
+173.7 "Scythe Kick" sync /:Shiva:4DE3:/
+181.9 "--Untargetable--"
+193.2 "Shattered World" sync /:Shiva:4DE9:/
+
+# Intermission. Fixed length.
+209.4 "Heart Asunder" sync /:Mothercrystal:4E12:/
+218.1 "Stoneskin" sync /:Earthen Aether:4DEF:/
```

Comment:
Can you comment this sync out, as this ability doesn't always fire?

Reply:
Oops, totally overlooked that! I'm actually a little surprised I never noticed the missed sync, since I'm always on an interrupt role while running this.